### ISSUE TITLE
Brief Introduction

### ISSUE THREAD

**Issue by weikee94** [chiamtc/rt#1]
Kindly add some brief explanation about this repo. 
Thanks.

**Comment by chiamtc** [comment]
My thesis topic - project management tool. Something similar to Jira (made by Atlassian), PivotalTracker and other numerous tools out there.

**Comment by weikee94** [comment]
Like Trello the one we used? Good!

**Comment by chiamtc** [comment]
Yes, but with more features for enterprise level. This is for thesis, so, mostly it isn't beautiful in design.

**Comment by weikee94** [comment]
When it will release? Cant wait to use it!

**Comment by chiamtc** [comment]
Not sure it will be released, maybe during this semester since I need users to test it and give feedback. You willing to help to test it if I publish it?

**Comment by weikee94** [comment]
Yeah, I am happy to give feedback to enhance user experience!

**Comment by chiamtc** [comment]
deployed but need to implement usability tool (soon). Link found in readme.

**Comment by weikee94** [comment]
There got underline space when I click RE TOOL.

**Comment by chiamtc** [comment]
what do you mean by underline? I thought the link in the readme description?

**Comment by chiamtc** [comment]
I saw your testing account and project..

**Comment by weikee94** [comment]
http://createretool.com/RT/#/home there are a RF on the left hand side (navigation) when you hover there have underline space. Haha yeah testing and play around and provide you feedback and make improvement.

**Comment by chiamtc** [comment]
oh, the margin thing which I will never be able to fix..

**Comment by weikee94** [comment]
I will help to look at it no worry mate XD

**Comment by chiamtc** [comment]
i will implement the testing tool soon..like how long user will take to create a backlog, task, assigning people or etc...

**Comment by chiamtc** [comment]
do tell me in details about the usability of using this tool to manage each use case (backlog) in sprint like ease of use, track, find, update. Assigning task (inside each backlog) to people. Then, after sprint completed, the review of each backlog with client.
